app/src/main/java/org/simple/clinic/bp/entry/ConfirmRemoveBloodPressureDialog.kt

```diff
@@ -54,6 +58,8 @@ class ConfirmRemoveBloodPressureDialog : AppCompatDialogFragment() {
   @Inject
   lateinit var controller: ConfirmRemoveBloodPressureDialogController
 
+  private var removeBloodPressureListener: RemoveBloodPressureListener? = null
```

Comment:
Can this ever be `null`? We throw an error when the activity has not implemented the interface, so this could be a `lateinit var`, right?

Reply:
This can become `null` during a configuration change. In such cases, we have to replace the null check with a `::property.isInitialized` check instead.